### PR TITLE
Supernode: artworkreg unit test

### PR DESCRIPTION
**Pull request by ashadi-cc**
- node unit test 
- task unit test

### CODE REVIEW

File: supernode/services/artworkregister/node.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ func (nodes Nodes) ByID(id string) *Node {
 func (nodes *Nodes) Remove(id string) {
 	for i, node := range *nodes {
 		if node.ID == id {
-			*nodes = append((*nodes)[:i], (*nodes)[:i+1]...)
```
Comment (ashadi-cc) [inline supernode/services/artworkregister/node.go:36]: @levkoburburas 
I found a bug on this line. actually the old code adding node instead remove node from nodes list. Let me know if i am wrong

**Review by Dicklesworthstone — APPROVED** [review]

File: supernode/services/artworkregister/node.go (hunk 1 above)
Comment (alonlong) [reply supernode/services/artworkregister/node.go:36]: the code is refactored, need u help to update the unit test for the p2p module in the future

Comment (alonlong) [inline supernode/services/artworkregister/node.go:36]: did u do test for it?

Comment (ashadi-cc) [reply supernode/services/artworkregister/node.go:36]: I did test it. its adding duplicate node at the end instead remove node

Comment (ashadi-cc) [reply supernode/services/artworkregister/node.go:36]: ok sure

Comment (alonlong) [reply supernode/services/artworkregister/node.go:36]: yeah, the refactored code is different, and i have unit tests for it, it works well, good catch

**Review by alonlong — APPROVED** [review]

**Comment by pjenys0** [comment]
Close due to outdated.

**Pull request closed without merging**